src/config/outputs/shell/logger/drain.rs

```diff
@@ -14,9 +14,22 @@ use lazy_static::lazy_static;
 
 const BUFFER_SIZE: usize = 32;
 
+pub struct ShellBuffer {
+    pub buffer: VecDeque<(Level, String)>,
+    pub updated: bool,
+}
+
+impl ShellBuffer {
+    pub fn new() -> Self {
+        Self {
+            buffer: VecDeque::with_capacity(BUFFER_SIZE),
+            updated: true,
```

Comment:
Looks great! But before you moved this to a struct, it used to be `false`, is this intentional change?

Reply:
Yes, just to make sure when everything starts and for example a the only used widget is the Logger widget and logs are empty the widget gets it first draw.
Text widget also starts with true.